package.json

```diff
@@ -18,11 +18,12 @@
     "@babel/runtime": "^7.6.2",
     "@rails/webpacker": "3.5",
     "babel-loader": "^8.0.6",
+    "foundation-sites": "^6.6.3",
```

Comment:
`foundation-rails` has already been added to the Gemfile

Reply:
deleted